garage/experiment/experiment_wrapper.py

```diff
@@ -183,11 +193,12 @@ def run_experiment(argv):
     snapshotter.snapshot_gap = args.snapshot_gap
     logger.push_prefix("[%s] " % args.exp_name)
 
-    if args.resume_from is not None:
-        data = joblib.load(args.resume_from)
-        assert 'algo' in data
-        algo = data['algo']
-        algo.train()
+    if args.resume_from_dir is not None:
+        with LocalRunner() as runner:
```

Comment:
maybe this is the form we should encourage using in the example?

Reply:
I think so. This is the most convenient way to resume an experiment. The programmatical way is more advanced in that it allows more flexibility.

Reply:
okay, can you use this form in the example then?

Reply:
Yes. I put it as recommended in the examples.